dev/integration_tests/web/lib/null_enabled_api.dart

```diff
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.12
-
-void methodThatAcceptsNonNull(int x) {
-  print(x + 2);
+void methodThatAcceptsNonNull(int? x) {
+  print(x! + 2);
```

Comment:
Why the added `?` and `!` here?

Reply:
We are trying to pass null here!

Reply:
<img width="840" alt="image" src="https://user-images.githubusercontent.com/56479665/121282730-9fe51580-c8f7-11eb-93fb-e9393e9610ca.png">

@goderbauer we need to pass null here.

Reply:
I'm confused. Isn't the point of the function that the method only accepts non-null values? In other words, just removing the comment `// @dart = 2.12` should be enough after reverting these changes.

Reply:
Hello @shihaohong 
here we need to pass null value. So how should I deal with this?

Reply:
@Abhishek01039 can you explain why we need to pass a null value here?

Reply:
@Piinks we need to assert here but I think we can remove this file `dev/integration_tests/web/lib/null_assert_main.dart`